File: Atcoder/abc116/b.py
s = int(input())
S = [s]
#print(S)
N = 1000000
for i in range(N):
	if s%2 == 0:
		s = s//2

		#print(s)

		if s in S:
			ans = i+2
			break
		S.append(s)
	else:
		s = 3*s+1

		#print(s)

		if s in S:
			ans = i+2
			break
		S.append(s)
#print(S)
print(ans)

File: Atcoder/abc116/b.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
	int ans;
	int s;
	cin >> s;
	set<int> st;
	int x = s;
	while(st.count(x) ==0){
		st.insert(x); //集合の中にxを入れる
		if (x%2 ==0)
		{
			x/=2;
		}
		else{
			x = x*3 + 1;
		}
	}
	ans = st.size()+1;
	cout << ans << endl;
}

